### PR TITLE
Move include_fbs from butte_build to butte.

### PR DESCRIPTION
**Pull request by mxpv**
Typically `butte-build` crate is expected to be referenced at build time to
generate code. However in current implementation it needs to be
referenced at runtime as well in order to use `include_fbs` macro.
This PR moves `include_fbs` macro to `butte` crate, so no extra dependency
will be required.

Signed-off-by: Maksym Pavlenko <pavlenko.maksym@gmail.com>

### CODE REVIEW

**Comment by mxpv** [comment]
These compiler/clippy warnings are not related to changes in this PR.
But I went ahead to fixed them anyway.

**Comment by cpcloud** [comment]
@mxpv Thanks!

**Comment by mxpv** [comment]
@cpcloud one more for generated code.

**Comment by mxpv** [comment]
Ok, I've fixed clippy warnings in nightly.
But now it complains about `matches!` in 1.41 (which was introduced in 1.42).
Is it ok to bump minimum version from 1.41 to 1.42?

**Comment by cpcloud** [comment]
> Is it ok to bump minimum version from 1.41 to 1.42?

Yup

File: butte-build/src/codegen.rs [hunk 1]
```diff
@@ -275,7 +275,10 @@ mod to_default_value_tests {
                 let value = ast::DefaultValue::Scalar($kind(rust_value));
                 let ty =
                     to_type_token(None, &ir::Type::$butte_ir_type, &quote!(), &quote!(), false);
-                let result = to_default_value(&ty, &value).to_string();
+                // Hack: there is no control to make TokenStream::ToString() print tokens without
+                // spaces in between (except running `cargo fmt`), so we're doing replace at the
+                // end to get properly formatted string (e.g. `- 123` -> `-123`).
+                let result = to_default_value(&ty, &value).to_string().replace("- ", "-");
```
Comment (cpcloud) [inline butte-build/src/codegen.rs:282]: Oh, well that's fun.

**Comment by cpcloud** [comment]
@mxpv Any chance you'd be able to describe your use case here? Really just curious.

**Comment by cpcloud** [comment]
Use case for butte itself, not this PR :)

**Comment by mxpv** [comment]
Hope https://github.com/butte-rs/butte/pull/84/commits/2e6b21b770fcc848265099b9f3d8a5f37b2680ae is the last one. 

> Any chance you'd be able to describe your use case here? Really just curious.

Oh, I'm just experimenting. Mainly this is interesting because I'm not required to carry `flatc` to generate schemas from `build.rs` :)

**Comment by cpcloud** [comment]
LGTM! Merging, thanks.

**Pull request merged**